request_logging/decorators.py

```diff
@@ -0,0 +1,10 @@
+from functools import wraps
+
+
+def no_logging(func):
```

Comment:
It would be good if this accepted a message that can be logged in place of the content. We should have a test to prove it works as well.